core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/copy-kubeconfig.yml

```diff
@@ -17,6 +17,24 @@
       become: false
       include_vars:
         file: "{{ vault_location }}/kubernetes-secrets.yml"
+ 
+    - name: Create kubeconfig file for local usages
+      delegate_to: localhost
+      become: false
+      template:
+        src: kubeconfig.j2
+        dest: "{{ vault_location }}/../admin.conf"
```

Comment:
Don't we want to put it in `~/.kube/config`?

Reply:
I would prefere to put it in build directory since in one container we can have more than one cluster build. I was talking about it with @to-bar and he aggred. Moreover, I created ticket #1650 because, we don't want to use all the time vault_location variables and right now we have to. We should have seperated variable: "build_location"

Reply:
You are right, inventory dir it is what I was looking for, thanks :) #1650 to close
@atsikham  As we already talked, we can have two cluster in one container.